fseai/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
```

Comment:
We don't need these pointless comments in the starting of every file. I said `pointless` to make it clear that these messages does not give anything to the reader. It's only recommended to add such dosctrings at the top of the file when you need to explain what it is in the file and you cannot explain that as a docstring of a function or a class which is in the file